features/performance/assets/mc-pre-boot-worker-tuning.yaml.in

```diff
@@ -0,0 +1,44 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+    labels:
+        machineconfiguration.openshift.io/role: worker-rt
+    name: 11-rt-kernel-worker-rt
+spec:
+    kernelArguments:
+        - 'non_iso_cpumask=00000005'
```

Comment:
I think this needs to be calculated based on what we set in `isolcpus` or what does this mask mean?

Reply:
Agreed, a comment would be most welcome